Description: Tweak D-Bus service activation configuration file:
 * log wpa_supplicant messages to syslog
 * activate control socket interface so that wpa_cli can be used by D-Bus
   activated wpa_supplicant daemon
Author: Kel Modderman <kel@otaku42.de>
---
--- a/wpa_supplicant/dbus/fi.epitest.hostap.WPASupplicant.service
+++ b/wpa_supplicant/dbus/fi.epitest.hostap.WPASupplicant.service
@@ -1,4 +1,4 @@
 [D-BUS Service]
 Name=fi.epitest.hostap.WPASupplicant
-Exec=/sbin/wpa_supplicant -u
+Exec=/sbin/wpa_supplicant -u -s -O /var/run/wpa_supplicant
 User=root
--- a/wpa_supplicant/dbus/fi.w1.wpa_supplicant1.service
+++ b/wpa_supplicant/dbus/fi.w1.wpa_supplicant1.service
@@ -1,4 +1,4 @@
 [D-BUS Service]
 Name=fi.w1.wpa_supplicant1
-Exec=/sbin/wpa_supplicant -u
+Exec=/sbin/wpa_supplicant -u -s -O /var/run/wpa_supplicant
 User=root
